src/filters/dasher.c

```diff
@@ -2840,6 +2841,24 @@ static void dasher_add_descriptors(GF_List **p_dst_list, const GF_PropertyValue
 	}
 }
 
+static void dasher_set_inband_event(GF_DashStream *ds) {
+	GF_MPD_Inband_Event *nielsen_event;
+	GF_MPD_Inband_Event *custom_event;
+	if(ds->stream_type == GF_STREAM_AUDIO) {
+		GF_SAFEALLOC(custom_event, GF_MPD_Inband_Event);
+		custom_event->scheme_id_uri = gf_strdup("https://aomedia.org/emsg/ID3");
+		custom_event->value = gf_strdup("www.geniussports.com:id3:v1");
```

Comment:
If this is a pull request into gpac I don't think we can use geniussports in the namespace

Reply:
No, this is into a dev branch for internal use. The actual fix for this is quite more elaborate, requiring to collect the scheme and value from the packets entering the `dasher` for the audio track and then creating the corresponding entries in the manifest.

For now, this flag unblock us for running the packager.

Reply:
@sebastianBD95 why did you choose `www.geniussports.com:id3:v1` for the value? We could use `https://aomedia.org/emsg/ID3` again to indicate it's a generic ID3 tag.

Reply:
set it to`https://aomedia.org/emsg/ID3`